service/collector/elb.go

```diff
@@ -12,23 +12,23 @@ import (
 )
 
 const (
-	ELBLabel = "elb"
+	labelELB = "elb"
 )
 
 const (
-	StateOutOfService = "OutOfService"
+	stateOutOfService = "OutOfService"
 )
 
 var (
 	elbsDesc *prometheus.Desc = prometheus.NewDesc(
-		prometheus.BuildFQName(Namespace, "", "elb_instance_out_of_service_count"),
+		prometheus.BuildFQName(namespace, "", "elb_instance_out_of_service_count"),
```

Comment:
Should we do this? Of course using a constant. Just showing the result for a discussion basis here. 

```suggestion
		prometheus.BuildFQName(namespace, "elb", "instance_out_of_service_count"),
```

Reply:
I like it. Would make it more consistent for the next metric we add.